apps/hostel/schemas.py

```diff
@@ -10,7 +12,7 @@ class HostelListOut(BaseOutSchema):
     street: str
     build: str
     commandant_id: int
-    commandant_full_name: str
+    commandant_full_name: Dict[str, str]
```

Comment:
Should be an embedded schema, that describes this structure

Reply:
example?

Reply:
fixed